cmake/Modules/CheckAVX2Support.cmake

```diff
@@ -4,7 +4,7 @@
 #
 # The MIT License
 #
-# Copyright (c) 2018 TileDB, Inc.
+# Copyright (c) 2020 TileDB, Inc.
```

Comment:
Do we want all the files that have a single year be converted to `2017-2020`?

Reply:
In a scenario like this, where it is currently `2018`, should we convert it to `2018-2020`, or just do a blanket `2017-2020` for everything?

Reply:
Sorry, ideally it would be `2018-2020`.

Reply:
Also is adding a space more common?  I think see more "2018 - 2020" than "2018-2020".  Not that it matters greatly but consistency is nice.

Reply:
Easy, will do.

Reply:
Yeah I saw that in @eddelbuettel changes in TileDB-R. Either way is fine, I agree on consistency.

Reply:
That was only two files so I could also change easily over there if we prefered the 'no whitespace' variant.

I think decades ago I looked at some GNU project and they seem to use a space.  So now I picked two random repos (Emacs, R) and saw two different forms even with double-hyphen.  Which makes me less sure I knew what my preference was based on ...

Reply:
Sounds good.  I'll adjust the R package accordingly including the two with the spaces :)